backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/FRBRExpression.java

```diff
@@ -0,0 +1,108 @@
+package de.bund.digitalservice.ris.norms.domain.entity;
+
+import de.bund.digitalservice.ris.norms.utils.NodeParser;
+import de.bund.digitalservice.ris.norms.utils.exceptions.MandatoryNodeNotFound;
+import java.util.Optional;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+
+/** Class representing the akn:FRBRExpression */
+@Getter
+@SuperBuilder(toBuilder = true)
+public class FRBRExpression extends FRBR {
+
+  private static final String VALUE_ATTIBUTE = "value";
+
+  public FRBRExpression(final Node node, final String normEli) {
+    super(node, normEli, FRBRExpression.class.getSimpleName());
+  }
+
+  /**
+   * Retrieves the UUID of the previous version.
+   *
+   * @return the uuid
+   */
+  public Optional<UUID> getFRBRaliasPreviousVersionId() {
+    return NodeParser.getNodeFromExpression("./FRBRalias[@name='vorherige-version-id']", getNode())
+        .map(m -> m.getAttributes().getNamedItem(VALUE_ATTIBUTE).getNodeValue())
+        .map(UUID::fromString);
+  }
+
+  /**
+   * Sets the FRBRalias for the previous version id.
+   *
+   * @param uuid the new uuid
+   */
+  public void setFRBRaliasPreviousVersionId(final UUID uuid) {
+    NodeParser.getNodeFromExpression("./FRBRalias[@name='vorherige-version-id']", getNode())
+        .orElseGet(
+            () -> {
+              final Element newElement =
+                  getNode().getOwnerDocument().createElement("akn:FRBRalias");
+              newElement.setAttribute("eId", "meta-1_ident-1_frbrexpression-1_frbralias-1");
```

Comment:
we have `Norm::createElementWithEidAndGuid` but I don't have an idea how to best use it here (or in other classes for parts of norms). But it could simplify the code here quite a bit

Reply:
We could for sure create later on a factory class to create different kind of nodes by passing the necessary info like parent node, tag name and the map list of attributes+values. But for now is in my opinion overengineered

Reply:
yes, sure, for now it works and we can keep it like it is :)